crates/node/core/src/args/static_files.rs

```diff
@@ -42,6 +46,10 @@ pub struct StaticFilesArgs {
     /// the node has been initialized, changing this flag requires re-syncing from scratch.
     #[arg(long = "static-files.transaction-senders")]
     pub transaction_senders: bool,
+
+    /// Storage account changesets in static files
```

Comment:
```suggestion
    /// Store account changesets in static files.
    
    /// When enabled, account changesets will be written to static files on disk instead of the
    /// database.
    ///
    /// Note: This setting can only be configured at genesis initialization. Once
    /// the node has been initialized, changing this flag requires re-syncing from scratch.
```